website/docs/checkaction.md

```diff
@@ -1,4 +1,5 @@
-checkaction - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+checkaction - What should happend when an assertion fails (D Lang)
```

Comment:
- [ ] The `checkaction -` is redundant. We use filename as a title then you can start right away with a description.
- [ ] It's important to be more explicit. Instead of `(D Lang)` add: _This option applies only to **D Programming Language**_